### ISSUE TITLE
Validation of BIDS2NDA output

### ISSUE THREAD

**Issue by mtnhuck** [INCF/BIDS2NDA#11]
Hi,
    I ran bids2nda without any issues and it generated a image03.txt. This is the same dataset that passed the bids validator I mentioned in a previous issue. I ran the NDA Validator and Upload tool, which ran but provides quite a few errors, all specifically related to missing data in fields for non-DICOM images.

### As best as I can tell the
fields missing data (left) map onto the following json fields (right):
_photomet_interpret PhotometricInterpretation
experiment_id ProcedureStepDescription
image_orientation ImageOrientationPatient
scanner_software_versions_pd SoftwareVersions_
_image_slice_thickness SliceThickness_ -> This one seem to be present in some of the .json files and not others (present for all functional/anatomical/fmap phasediff runs but not for fmap magnitude. I'm assuming the last part may be a bug on the conversion to BIDS but image_slice_thickness doesn't seem to map onto SliceThickness from the json file for any of the files.

### Couldn't find these in the json files (or not exactly sure what they
are looking for):
_extent5_type
image_unit5
image_extent5
image_resolution5_

Would there be a way to potentially pull data formats directly from NDA as mentioned in https://github.com/INCF/BIDS2NDA/issues/8? 

Thanks in advance and please let me know if I can provide more information etc.
Jeremy

**Comment by chrisfilo** [comment]
Those fields seems to be added more recently, thus breaking the converter.
Pulling a never version of image03.txt would help testing, but work needs
to be done to implement new mapings from BIDS to NDA (for the new fileds).
Would you like to help implementing those?

Best,
Chris

**Comment by mtnhuck** [comment]
Of course, I’m happy to help however I can. Just let me know what I can do
to move things in the right direction.
Cheers
Jeremy

**Comment by chrisfilo** [comment]
If you could send a Pull Request on GitHub with modifications adding at
least some of the missing mappings that would be greatly appreciated.

Best,
Chris

**Comment by agrethe** [comment]
Hello, 

Just following up on this to see if there are plans to update bids2nda to be compliant with the image03 format?  I have the same issues with missing data for the above fields, as well as fields that are optional for image03.csv but generate warnings with the NDA validator tool because the columns don't exist. 

Thank you --
Amanda
